miio/g1vacuum.py

```diff
@@ -0,0 +1,281 @@
+import logging
+from typing import Any, Dict
+from datetime import timedelta
+from enum import Enum
+import click
+from .click_common import EnumType, command, format_output
+from .miot_device import DeviceStatus, MiotDevice
+
+_LOGGER = logging.getLogger(__name__)
+MIJIA_VACUUM_V2 = "mijia.vacuum.v2"
+
+MIOT_MAPPING = {
+    MIJIA_VACUUM_V2: {
+    # https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:vacuum:0000A006:mijia-v1:1
+    "battery": {"siid": 3, "piid": 1},
+    "charge_state": {"siid": 3, "piid": 2},
+    "error_code": {"siid": 2, "piid": 2},
+    "state": {"siid": 2, "piid": 1},
+    "fan_speed": {"siid": 2, "piid": 6},
+    "operating_mode": {"siid": 2, "piid": 4},
+    "mop_state": {"siid": 16, "piid": 1},
+    "water_level": {"siid": 2, "piid": 5},
+    "main_brush_life_level": {"siid": 14, "piid": 1},
+    #  "main_brush_life_time": {"siid": 14, "piid": 2},
```

Comment:
Are these commented out properties responding with correct values? If yes, could these be readded and could you also follow the same API as defined in `vacuumcontainers.py` (like a recent roidmi one here: https://github.com/rytilahti/python-miio/pull/1072/files#diff-c42fe5b3b81b793b81d9fef91c215000978491bc01cb2b682e9d225ee8c6f183R463)?

I'm sorry that the API is a mess, and that's why I'm slowly trying to move this library towards having stricter API constraints wrt. naming to make it simpler for downstreams to support vacuums from all supported vendors.

Reply:
The problem obviously was a timeout error when getting too many properties. 

I followed the Roidmi example and split the stati in status, consumable_status and cleaning_summary.
But it seems that the totals in cleaning_summary always return 0 :(

I hope that is better than before.